CRM/Event/Form/Registration/Register.php

```diff
@@ -588,15 +592,8 @@ public static function buildAmount(&$form, $required = TRUE, $discountId = NULL,
       $form->add('hidden', 'priceSetId', $priceSetID);
 
       // CRM-14492 Admin price fields should show up on event registration if user has 'administer CiviCRM' permissions
-      $adminFieldVisible = FALSE;
-      if (CRM_Core_Permission::check('administer CiviCRM data')) {
-        $adminFieldVisible = TRUE;
-      }
-
-      $hideAdminValues = TRUE;
-      if (CRM_Core_Permission::check('edit event participants')) {
-        $hideAdminValues = FALSE;
-      }
```

Comment:
Sadly not the *most* overcomplicated method I've seen of assigning a boolean value to a variable.

Reply:
@colemanw there is competition!

Reply:
It could have first assigned it to an undeclared class variable :)

Reply:
@colemanw there is that